containers/dotnet/definition-manifest.json

```diff
@@ -7,7 +7,11 @@
 		"tags": [
 			"dotnet:${VERSION}-${VARIANT}"
 		],
+		"architectures": ["linux/amd64", "linux/arm64"],
 		"variantTags": {
+			"6.0-focal": [
```

Comment:
This is not needed. It's covered by the variants and the primary `tag` property.  We don't add dotnetcore to new versions of .NET given that brand is gone. The others are there for back compat post change.

The .NET image defaults to debian bullseye, which is why this is also defaulting to bullseye for pure "6.0".